httping: add page (#2768)
tldr-pages/tldr: pages/common/httping.md

# httping

> > Measure the latency and throughput of a webserver.

Reviewer: Perhaps this will read better?
```suggestion
> Measure the latency and throughput of a web server.
```

> - Ping the webserver on `host`:

Reviewer: Perhaps this will read better?
```suggestion
- Ping the web server on `host`:
```

> `httping -g http://localhost/`

Reviewer: Could we surround `http://localhost/` in `{{` and `}}` please?

Author: updated, thank you:)

> - Ping the webserver on `host` and `port`:

Reviewer: Perhaps this will read better?

```suggestion
- Ping the web server on `host` and `port`:
```

Reviewer: I think you missed this one, @thiswind :P

> `httping -h localhost -p 1000`

Reviewer: Same here with `localhost` and `1000`.

Author: updated :)

> - Ping the webserver on `host` using an SSL connection:

Reviewer: Perhaps this will read better?

```suggestion
- Ping the web server on `host` using a TLS connection:
```

Author: thank you, and I will do better next time :)

> `httping -l -g https://localhost/`

Reviewer: Same here with `https://localhost/`.

Author: updated :)

> - Ping the webserver on host 'localhost' using the Basic HTTP Authentication:

Reviewer: Perhaps this will read better?

```suggestion
- Ping the web server on host 'localhost' using HTTP basic authentication:
```

> `httping -g http://localhost/ -U username -P password`

Reviewer: Same here with `http://localhost/`, `username`, and `password`.

Author: updated :)
